packages/react/focus-scope/src/FocusScope.tsx

```diff
@@ -128,7 +136,7 @@ const FocusScope = React.forwardRef((props, forwardedRef) => {
           const unmountEvent = new Event(AUTOFOCUS_ON_UNMOUNT, EVENT_OPTIONS);
           container.dispatchEvent(unmountEvent);
           if (!unmountEvent.defaultPrevented) {
-            focus(PREVIOUSLY_FOCUSED_ELEMENT ?? document.body, { select: true });
+            focus(previouslyFocusedElement ?? document.body, { select: true });
```

Comment:
I think this is the only place where we really need `unmountAutoFocusElement ?? document.activeElement ?? document.body`